common/eq_packet_structs.h

```diff
@@ -5358,7 +5358,7 @@ struct ItemPreview_Struct
 struct MercenaryList_Struct {
 /*0000*/	uint32	MercID;				// ID unique to each type of mercenary (probably a DB id)
 /*0004*/	uint32	MercType;			// From dbstr_us.txt - Apprentice (330000100), Journeyman (330000200), Master (330000300)
-/*0008*/	uint32	MercSubType;		// From dbstr_us.txt - 330020105^23^Race: Guktan<br>Type: Healer<br>Confidence: High<br>Proficiency: Apprentice, Tier V...
```

Comment:
This file got hit with a find/replace

Can revert it via `git checkout common/eq_packet_structs.h master`